From 7e53f0162c69828138418eca94fe1970aa029c5b Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 19 May 2020 14:30:11 +0800
Subject: [PATCH] ARM: dts: rockchip: Set reboot-freq to 1.2GHz for
 rv1126-evb-ddr3-v10-tb

Same as boot freq, prevent cpufreq drop to 816MHz which will make vdd_arm
drop to 0.725v.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I1c0c95b2ec293e41fdfb68474d62b7aca568fe96
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
index 62358c6894fc..b7e66776b11f 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v10-tb.dts
@@ -54,6 +54,7 @@
 };
 
 &cpu0_opp_table {
+	rockchip,reboot-freq = <1200000>;
 	/delete-node/ opp-1296000000;
 	/delete-node/ opp-1416000000;
 	/delete-node/ opp-1512000000;
-- 
2.35.3

